scripts/examples/webos_example.sh

```diff
@@ -15,35 +15,55 @@
 #    limitations under the License.
 #
 
+SCRIPT_DIR=$(dirname "$0")
```

Comment:
Could we create a target in [scripts/build/builders](https://github.com/project-chip/connectedhomeip/tree/master/scripts/build/builders) for webos and use `scripts/build/build_examples.py` for building?

Reply:
I think we could have a script for bootstrap if needed (all the unpack logic), however the rest of the logic of `gn gen` and `ninja -C` should be trivially convertable to build_examples and would make build instructions standard.